lib/galaxy/tools/parameters/dynamic_options.py

```diff
@@ -158,8 +163,20 @@ def get_dependency_name(self):
         return self.ref_name
 
     def filter_options(self, options, trans, other_values):
+        def _add_meta(meta_value, m):
+            if isinstance(m, list):
+                meta_value |= set(m)
+            elif isinstance(m, dict) or isinstance(m, OrderedDict):
```

Comment:
```suggestion
            elif isinstance(m, dict):
```
OrderedDict is an instance of dict.